data/deep missions.txt

```diff
@@ -177,22 +182,22 @@ mission "Deep: Kraz Convoy"
 	to offer
 		"mystery cube" >= 10
 		or
-			random < 30
+			random < 20
 			and
 				"deep convoy" == 1
-				random < 40
+				random < 30
 			and
 				"deep convoy" == 2
-				random < 50
+				random < 40
 	on offer
 		conversation
-			`A Deep Security officer approaches you when you enter the spaceport and asks if you would be able to do a job for the Deep. "We're short on Deep Security ships at the moment, but we need three Behemoths escorted to <stopovers> where they will pick up cargo, then escorted back here by <date>."
+			`A Deep Security officer approaches you when you enter the spaceport and asks if you would be able to do a job for the Deep. "We're short on Deep Security ships at the moment, but we need three Behemoths escorted to <stopovers>. After they pick up the cargo, bring them back here by <date>."
 			choice
 				`	"Sure, I can do that."`
 				`	"Sorry, but I will not be traveling in that direction any time soon."`
-					decline
+					defer
 			
-			`	"Much appreciated, Captain. And for your well being as well as that of the cargo's, you must know that there may be pirates waiting in Algieba, Cor Caroli, and Wei. Behemoths are tough ships so you should be able to travel through those systems and not slow yourself down by taking alternative routes."`
+			`	"Much appreciated, Captain. We expect there will be pirates waiting in Algieba, Cor Caroli, and Wei, but Behemoths are tough ships so you shouldn't need to slow yourself down by taking an alternative route."`
```

Comment:
I generally prefer the original way I wrote this; this specific officer will pop up again (although named next time [psst. it's Paris]), and the (what I guess you could say) "strange" way I wrote this was completely intentional.